doc(plugin):Add info about cachegroup.test and cachegroup.name (#3459)
webpack/webpack.js.org: src/content/plugins/split-chunks-plugin.md

W> When files paths are processed by webpack, they always contain `/` on Unix systems and `\` on Windows. That's why using `[\\/]` in `{cacheGroup}.test` fields is necessary to represent a path separator. `/` or `\` in `{cacheGroup}.test` will cause issues when used cross-platform.

> W> Passing an entry name to `{cacheGroup}.test` is no longer allowed from Webpack 5.

Reviewer: ```suggestion
W> Passing an entry name to `{cacheGroup}.test` is no longer allowed from webpack 5.
```

Author: Resolved

> W> Passing an entry name to `{cacheGroup}.test` is no longer allowed from webpack 5.

Reviewer 2: ```suggestion
W> Since webpack 5, passing an entry name to `{cacheGroup}.test` and using a name of an existing chunk for `{cacheGroup}.name` is no longer allowed.
```

> W> Using a name of an existing chunk is not longer allowed for `{cacheGroup}.name` (it will throw an error).

Reviewer 3: ```suggestion
W> Using a name of an existing chunk is no longer allowed for `{cacheGroup}.name` (it will throw an error).
```

Author: Thanks for pointing out :)

> W> Using a name of an existing chunk is no longer allowed for `{cacheGroup}.name` (it will throw an error).
> 

Reviewer 2: ```suggestion
```
